src/components/Common/FilePreviewDialog.tsx

```diff
@@ -312,7 +312,7 @@ const FilePreviewDialog = (props: FilePreviewProps) => {
             <div className="mb-2 flex flex-col items-start justify-between md:flex-row">
               <div>
                 <TooltipComponent content={fileName}>
-                  <p className="text-2xl font-bold text-gray-800 truncate">
+                  <p className="text-2xl font-bold text-gray-800">
                     {fileNameTooltip}
                   </p>
```

Comment:
let's move this to the place where `DialogTitle` is present, and remove "File preview" from `DialogTitle`

The created on date just below this can be moved inside the DialogHeader (below the DialogTitle) as well!